src/main/resources/swagger/api-docs.yaml

```diff
@@ -1712,6 +1715,67 @@ paths:
         500:
           description: Internal Server Error
 
+  /api/trial/manager/report:
+    post:
+      tags:
+        - Trial
+      operationId: createReportTrialProjects
+      summary: Create a Spreadsheet Report of free trial projects
+      responses:
+        200:
+          description: OK; report data spreadsheet is created
+          schema:
+            $ref: '#/definitions/SpreadsheetResponse'
+        403:
+          description: You must be a campaign manager to call this endpoint.
+        500:
+          description: Internal Server Error
+      security:
+        - googleoauth:
+          - openid
+          - email
+          - profile
+          - https://www.googleapis.com/auth/drive
+          - https://www.googleapis.com/auth/drive.file
+          - https://www.googleapis.com/auth/spreadsheets
+          - https://www.googleapis.com/auth/devstorage.full_control
+
+  /api/trial/manager/report/{spreadsheetId}:
+    put:
+      tags:
+        - Trial
+      operationId: updateReportTrialProjects
+      summary: Update a Spreadsheet Report of free trial projects
+      parameters:
+        - name: spreadsheetId
+          description: |
+            The spreadsheet id to update. The spreadsheet id can be found in the url of the link.
```

Comment:
I am a bit worried about users clobbering existing spreadsheets of theirs... maybe we should say something like
> This endpoint replaces the contents of the specified Google spreadsheet with the free trial report. The spreadsheetId can be found in the URL of the link. For example, in the link "https://docs.google.com/spreadsheets/d/long-random-string/edit", the string "long-random-string" is the "spreadsheetId" value. (You may wish to pre-create a blank spreadsheet to use this feature.)

Reply:
Yes, I have that task as a separate story (see https://broadinstitute.atlassian.net/browse/GAWB-2963). The current approach has Moran's blessing, as flawed as it is.